src/librustc_const_eval/diagnostics.rs

```diff
@@ -452,12 +452,14 @@ enum Method { GET, POST }
 
 
 E0297: r##"
+#### Note: this error code is no longer emitted by the compiler.
```

Comment:
@rust-lang/compiler This is an annoying (lack of a?) convention - is there any consensus?